app/services/Agents.scala

```diff
@@ -79,21 +82,25 @@ class Agents @Inject()(amiableConfigProvider: AmiableConfigProvider, lifecycle:
     )
   }
 
-  def refreshOldProdInstanceCount(): Unit = {
+  def refreshInstancesInfo(): Unit = {
```

Comment:
I could split this functions into 2 but that would mean querying the instances and AMIs twice at every refresh interval